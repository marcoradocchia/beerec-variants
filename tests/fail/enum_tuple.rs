use beerec_variants::Variants;

#[derive(Variants)]
pub enum Enum {
    Variant(usize, usize),
}

fn main() {}
