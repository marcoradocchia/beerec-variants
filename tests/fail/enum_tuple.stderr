error: Unsupported shape `unnamed fields`. Expected no fields.
 --> tests/fail/enum_tuple.rs:3:10
  |
3 | #[derive(Variants)]
  |          ^^^^^^^^
  |
  = note: this error originates in the derive macro `Variants` (in Nightly builds, run with -Z macro-backtrace for more info)
